config/data/index.js

```diff
@@ -0,0 +1,8 @@
+// Add data sources here
+const dataPlugin = (eleventyConfig) => { /* eslint-disable-line no-unused-vars */
```

Comment:
Do we need this file?

Reply:
Quite a story here: I created this file thinking I'd connect the related posts as a data source (which did not work), then I went to delete it, then I realized that someone is bound to eventually add data to 11ty config and wonder where to put it, so I was like "I might as well leave this for them"

So my answer would be "no but actually yes", but that's a loosely held opinion.